src/styles/blocks/feedback-form.css

```
@@ -6,6 +6,8 @@
 
 .feedback-form__title {
   margin-bottom: 0.4em;
+  margin-left: auto;
+  margin-right: auto;
```

```suggestion
  margin-inline: auto;
```

Можно сократить до одной строчки =)
Проверь, пожалуйста, что такое решение точно работает и, если оно тебе нравится, то прими правочку.

---

@solarrust, проверил, работает.